virttest/env_process.py

```diff
@@ -569,6 +571,12 @@ def preprocess(test, params, env):
     """
     error_context.context("preprocessing")
 
+    global _setuper_mgr
+    _setuper_mgr = test_setup.SetuperMgr(test, params, env)
```

Comment:
@luckyh, I agree this would enhance the way we perform test setup, still we use env in test_setup but one level up abstracted right ?

Reply:
IMO, `env` is designed for passing information across tests and all the test phases of each test (although it seems not support the former now), so the answer is yes for some setup procedure that really wants to inject something into `env`.

Reply:
Agreed.